test/NosCore.GameObject.Tests/Services/InventoryService/InventoryServiceTests.cs

```diff
@@ -219,7 +220,62 @@ public void MoveHalfSlotAndMergeThemWithOverflow()
             Assert.IsTrue((destinationItem?.ItemInstance?.Amount == 999) && (destinationItem.Slot == 1));
         }
 
-        //TODO RemoveItemAmountFromInventory
```

Comment:
You remove todo but it was here for the other one not for yours 😂 still need to be tested. Moreover yours will disapear for the load one